### ISSUE TITLE
Rethinking_2 environment.yml installs pymc and not pymc3

### ISSUE THREAD

**Issue by udibr** [pymc-devs/resources#168]
The notebooks in Rethinking_2 folder import pymc3 and not pymc this results in a missing module error.
I tried to manually change the notebook Chap_02 to use pymc but then the numeric results are wrong.
I finally did `pip install pymc3` and the notebook run flawlessly

**Comment by seanaedmiston** [comment]
Found same issue. Get no errors - just wrong numerical results (on the quadratic approximation stuff) when using pymc/main (was 4.0.0b).

**Comment by ericmjl** [comment]
Good catch, @seanaedmiston. Could you post a reprex (minimal reproducible example) for the devs on the main PyMC issue tracker, if not already done? That will help us iron out bugs that you all have awesomely found!
